### PR TITLE
Updated README.md file

### PR DESCRIPTION
**Pull request by Shiva-Sai-ssb**
## Changes proposed

 1. Added official website links to the tech stack images, rather than image links.
 2. Made changes to contribution steps of the project by adding two more steps which are helpful for beginners. I have added steps no. 7 and 8.

## Screenshots

Screenshot (51)

## Note to reviewers

 Personally, I felt steps no. 7 and 8 should be present.
 Open to Suggestions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,13 +93,24 @@ You can see the live demo at: https://linkshub.vercel.app
     ```sh
    npm run dev
    ```
-7. Push your changes into your branch
+
+7. Make your changes to the source code.
```
Comment (rupali-codes) [inline README.md:97]:
```suggestion
7. Make your changes 
```

**Review by rupali-codes — COMMENTED** [review]
We should remove source code ig, because there might people who want to add links or modify the docs

File: README.md [hunk 2]
```diff
@@ -86,13 +93,24 @@ You can see the live demo at: https://linkshub.vercel.app
...
+
+8. Stage your changes and commit:
+    ```sh
+   git add <filename>
+    ```
+
```
Comment (rupali-codes) [inline README.md:103]: Also, we should create a new step for committing the changes

`9. commit your changes`

Comment (Shiva-Sai-ssb) [reply README.md:103]: Ok thanks, I will do it.

File: README.md (hunk 1 above)
Comment (CBID2) [reply README.md:97]: I second this!

File: README.md [hunk 3]
```diff
@@ -86,13 +93,24 @@ You can see the live demo at: https://linkshub.vercel.app
...
+   ```
+9. Push your changes into your branch
    
     ```sh
    git push origin <your_branch_name>
    ```
 
-8. Create a [Pull Request](https://github.com/rupali-codes/LinksHub/compare) 
+10. Create a [Pull Request](https://github.com/rupali-codes/LinksHub/compare) 
    > Click _compare acorss forks_ if you don't see your branch
```
Comment (CBID2) [inline README.md:114]: Turn "acorss" into "across"

File: README.md [hunk 4]
```diff
@@ -86,13 +93,24 @@ You can see the live demo at: https://linkshub.vercel.app
...
    
     ```sh
    git push origin <your_branch_name>
    ```
 
-8. Create a [Pull Request](https://github.com/rupali-codes/LinksHub/compare) 
+10. Create a [Pull Request](https://github.com/rupali-codes/LinksHub/compare) 
    > Click _compare acorss forks_ if you don't see your branch 
 
 #### _Want to add your favorite Links into the Hub? make sure to follow [CONTRIBUTING guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)._
```
Comment (CBID2) [inline README.md:116]: Lowercase "Links" & "Hubs" (e.g., "links" and "hubs")

File: README.md (hunk 3 above)
Comment (Shiva-Sai-ssb) [reply README.md:114]: Sure, I will do it.

File: README.md (hunk 4 above)
Comment (Shiva-Sai-ssb) [reply README.md:116]: Sure, I will do it.

**Comment by Shiva-Sai-ssb** [comment]
Hey @rupali-codes, Made changes as suggested.

**Review by CBID2 — APPROVED** [review]
Looks great @Shiva-Sai-ssb! :) LGTM!

**Comment by Shiva-Sai-ssb** [comment]
> Looks great @Shiva-Sai-ssb! :) LGTM!

Thanks..

File: README.md [hunk 5]
```diff
@@ -86,16 +93,28 @@ You can see the live demo at: https://linkshub.vercel.app
...
+    ```sh
+   git add <filename>
+    ```
+
+9. Commit your changes
+    ```sh
+   git commit -m <your-commit-message>
+   ```
+
+9. Push your changes into your branch
```
Comment (rupali-codes) [inline README.md:109]:
```suggestion
10. Push your changes into your branch
```

**Review by CBID2 — COMMENTED** [review]
LGTM! 😄 👍🏾

**Review by CBID2 — APPROVED** [review]
Looks great! :)

**Comment by Shiva-Sai-ssb** [comment]
> Looks great! :)

Thanks. Hey @rupali-codes, could you review and merge it!

File: README.md [hunk 6]
```diff
@@ -106,7 +106,7 @@ You can see the live demo at: https://linkshub.vercel.app
    git commit -m <your-commit-message>
    ```
 
-9. Push your changes into your branch
+10. Push your changes into your branch
     ```sh
    git push origin <your_branch_name>
```
Comment (rupali-codes) [inline README.md:111]:
```suggestion
    git push origin <your_branch_name>
```

File: README.md [hunk 7]
```diff
@@ -106,7 +106,7 @@ You can see the live demo at: https://linkshub.vercel.app
...
    ```
```
Comment (rupali-codes) [inline README.md:112]:
```suggestion
    ```
```

**Review by rupali-codes — COMMENTED** [review]
Having issues

image

File: README.md [hunk 8]
```diff
@@ -108,7 +108,7 @@ You can see the live demo at: https://linkshub.vercel.app
 
 10. Push your changes into your branch
     ```sh
-   git push origin <your_branch_name>
+    git push origin <your_branch_name>
    ```
```
Comment (rupali-codes) [inline README.md]:
```suggestion
    ```
```

**Review by rupali-codes — COMMENTED** [review]
it's showing the file unexpectedly, just make this changes

**Comment by Shiva-Sai-ssb** [comment]
@rupali-codes, made this time.

Screenshot (56)

**Review by rupali-codes — APPROVED** [review]
thanks

**Comment by Shiva-Sai-ssb** [comment]
@rupali-codes, Thanks. Merge this one too!! #313

**Pull request merged**